language-plutus-core/bench/Bench.hs

```diff
@@ -66,10 +75,6 @@ main =
           largeTypeFile1 = BSL.readFile "test/types/tail.plc"
           largeTypeFile2 = BSL.readFile "test/types/verifyIdentity.plc"
           largeTypeFiles = (,,) <$> largeTypeFile0 <*> largeTypeFile1 <*> largeTypeFile2
```

Comment:
What's so large about those files? I'd call them small. Which I think is a great weakness of the current benchmarks, because as far as I see we do not bench anything large at all.

Reply:
:100:

Reply:
Do we have anything nontrivial we could do with them?

Reply:
Why don't we just get some bigger files? There are plenty in the test output of the plugin, for example.

Reply:
That's a good idea.